api/v1beta1/infraenv_types.go

```diff
@@ -98,6 +98,24 @@ type InfraEnvSpec struct {
 	// +kubebuilder:default=DiscoveryImageAlways
 	// +optional
 	IPXEScriptType IPXEScriptType `json:"ipxeScriptType"`
+
+	// The discovery kernel arguments to be passed during boot time.
```

Comment:
The k8s API convention is to start each of these comments with the property name.  So this would be "KernelArguments is a list of kernel arguments to pass..."

Same for Operation and Value below